app/frontend/Calendar/EditEvent/DialogHeader.svelte

```diff
@@ -279,9 +272,10 @@
   }
 
   async function onDeleteRemainder() {
+    let end = new Date(event.startTime);
     event.parentEvent.cancelEditing();
     event.cancelEditing();
-    await event.truncateRecurrence();
+    await event.setRecurrenceEndTime(end);
```

Comment:
`event` isn't the master here, and you probably want to delete the current instance too:
```suggestion
    event.parentEvent.cancelEditing();
    event.cancelEditing();
    await event.truncateRecurrencesFromHere();
```